### ISSUE TITLE
Bignum Fixnum deprecation warnings in Ruby 2.4

### ISSUE THREAD

**Issue by amingilani** [ctran/annotate_models#451]
I get this error whenever annotate autoruns on migrations:

```
/Users/gilani/.rvm/gems/ruby-2.4.0/gems/annotate-2.7.1/lib/annotate/annotate_models.rb:169: warning: constant ::Fixnum is deprecated
/Users/gilani/.rvm/gems/ruby-2.4.0/gems/annotate-2.7.1/lib/annotate/annotate_models.rb:169: warning: constant ::Fixnum is deprecated
/Users/gilani/.rvm/gems/ruby-2.4.0/gems/annotate-2.7.1/lib/annotate/annotate_models.rb:169: warning: constant ::Bignum is deprecated
/Users/gilani/.rvm/gems/ruby-2.4.0/gems/annotate-2.7.1/lib/annotate/annotate_models.rb:169: warning: constant ::Fixnum is deprecated
/Users/gilani/.rvm/gems/ruby-2.4.0/gems/annotate-2.7.1/lib/annotate/annotate_models.rb:169: warning: constant ::Bignum is deprecated
```

**Comment by lightbe** [comment]
I got the same error just now.

**Comment by morgoth** [comment]
It was fixed in https://github.com/ctran/annotate_models/commit/3fa1703e3e5e096fb14cdf05274a64aa1c22cbc6

Would be great to have new release with it

**Comment by lightbe** [comment]
Does that fix support Rake 12?

**Comment by rvirani1** [comment]
Also getting the same warnings

**Comment by jwg2s** [comment]
@ctran any chance we can get a minor release with this fix?

**Comment by cameronbourgeois** [comment]
This was fixed in release 2.7.2 https://github.com/ctran/annotate_models/releases/tag/v2.7.2

**Comment by ctran** [comment]
@cameronbourgeois, Thanks for noticing that.
